pandas/core/generic.py

```diff
@@ -2127,10 +2127,11 @@ def _repr_data_resource_(self):
         storage_options_versionadded="1.2.0",
         extra_parameters=textwrap.dedent(
             """\
-        engine_kwargs : dict, optional
-            Arbitrary keyword arguments passed to excel engine.
-    """
+            engine_kwargs : dict, optional
```

Comment:
Did the indentation need to change on this? Also likely another reason for the docstring failures

Reply:
I agree, this was probably something that went unnoticed, I'll fix it.